### PR TITLE
convert toMapData functions to DTO's and delete unused models

### PR DESCRIPTION
**Pull request by achour94**

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/model/ElementType.java [hunk 1]
```diff
@@ -30,6 +32,8 @@
 public enum ElementType {
     SUBSTATION(Substation.class, AbstractSubstationInfos::toData),
     VOLTAGE_LEVEL(VoltageLevel.class, AbstractVoltageLevelInfos::toData),
+    VOLTAGE_LEVEL_EQUIPMENTS(Connectable.class, AbstractVoltageLevelEquipmentsInfos::toData),
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/model/ElementType.java:35]: VOLTAGE_LEVELS_CONNECTABLES or CONNECTABLES ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
There are still toMapData methods for the ratioTapChanger, phaseTapChanger, current limits for the two windings transformers and lines ...

File: src/main/java/org/gridsuite/network/map/dto/bus/AbstractBusInfos.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package org.gridsuite.network.map.dto.bus;
```
Comment (AAJELLAL) [inline src/main/java/org/gridsuite/network/map/dto/bus/AbstractBusInfos.java:1]: You need to add copyright:
/**
 * Copyright (c) 2023, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

File: src/main/java/org/gridsuite/network/map/dto/bus/BusListInfos.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package org.gridsuite.network.map.dto.bus;
```
Comment (AAJELLAL) [inline src/main/java/org/gridsuite/network/map/dto/bus/BusListInfos.java:1]: The same thing here

File: src/main/java/org/gridsuite/network/map/dto/voltagelevelequipments/AbstractVoltageLevelEquipmentsInfos.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package org.gridsuite.network.map.dto.voltagelevelequipments;
```
Comment (AAJELLAL) [inline src/main/java/org/gridsuite/network/map/dto/voltagelevelequipments/AbstractVoltageLevelEquipmentsInfos.java:1]: Copyright

File: src/main/java/org/gridsuite/network/map/dto/voltagelevelequipments/VoltageLevelEquipmentsListInfos.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package org.gridsuite.network.map.dto.voltagelevelequipments;
```
Comment (AAJELLAL) [inline src/main/java/org/gridsuite/network/map/dto/voltagelevelequipments/VoltageLevelEquipmentsListInfos.java:1]: Copyright

**Pull request closed without merging**